clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/ui/testsResults/UTBotTestRunLineMarkerProvider.kt

```diff
@@ -51,7 +55,13 @@ class UTBotTestRunLineMarkerProvider : LineMarkerProvider {
                 return UTBotRunWithCoverageLineMarkerInfo(element, message, getStatusIcon(element))
             }
 
-            private fun getStatusIcon(element: PsiElement): Icon {
+            private fun isElementInTestFileGeneratedByUTBot(element: PsiElement) =
+                element.containingFile.virtualFile.localPath.let {
```

Comment:
Is this the same way we check the 'test' status in VSCode plugin?

Reply:
https://github.com/UnitTestBot/UTBotCpp/blob/32c0298667653f493d5c5543f60b527da61b2c11/vscode-plugin/src/codelens/codelensProvider.ts#L36

not exactly, in VSCode code lenses are placed in any file that are in tests directory. I think it is not quite good, because if you open any file and type in 'TEST' then code lenses for running utbot tests will appear, that's why I added an extra condition

Reply:
Not `lenses` but `gutter to run a test`.